README.md

```diff
@@ -178,13 +177,13 @@ config :mongoose_push, apns: [
    ]
  ]
  ```
-Analogically to `FCM` configuration, at top level we may specify named pools that have different configurations. For `APNS` this is specifically useful since Apple delivers different APS certificated for development and production use. As in `FCM`, `REST` client may select named pool by providing matching `:mode` in his `REST` request.
+Just like for `FCM`, at the top level we can specify the named pools that have different configurations. For `APNS` this is especially useful since Apple delivers different APS certificates for development and production use. The HTTP client can select a named pool by providing a matching :mode in the HTTP request.
 
 Each `APNS` pool may be configured by setting the following fields:
 * **cert** (*required*) - relative path to `APNS` `PEM` certificate issued by Apple. This certificate have to be somewhere in `priv` directory
 * **key** (*required*) - relative path to `PEM` private key for `APNS` certificate issued by Apple. This file have to be somewhere in `priv` directory
 * **pool_size** (*required*) - maximum number of used `HTTP/2` connections to google's service
-* **mode** (*either `:prod` or `:dev`*) - pool's mode. `REST` client may select pool used to push his notification by specifying matching option in his request
+* **mode** (*either `:prod` or `:dev`*) - pool's mode. The `HTTP` client may select pool used to push his notification by specifying matching option in the request
```

Comment:
The `HTTP` client may select a pool used to push notifications by specifying matching option in the request